inc/namespace.php

```diff
@@ -248,3 +253,36 @@ function override_wizard_styles() {
 	wp_register_style( 'altis-seo', plugin_dir_url( dirname( __FILE__ ) ) . 'assets/global-styles.css', [], '2021-06-04-5' );
 	wp_print_styles( 'altis-seo' );
 }
+
+/**
+ * Hide the social previews if Yoast Premium is not active.
+ */
+function hide_yoast_premium_social_previews() {
+	$screen = get_current_screen();
+
+	// Bail early if Yoast Premium is active or if we aren't on a post edit screen.
+	if ( is_yoast_premium() || $screen->base !== 'post' ) {
+		return;
+	}
+
+	/**
+	 * This targets the 6th and 7th components panel in the Yoast
+	 * sidebar, which corresponds to the Facebook and Twitter social
+	 * preview buttons. If Yoast ever adds more panels to this sidebar,
+	 * this will need to be updated.
+	 */
+	$styles = 'div.components-panel div:nth-child(6n) div.yoast.components-panel__body, div.components-panel div:nth-child(7n) div.yoast.components-panel__body {
```

Comment:
A good future update might be to ask if they can provide hooks to switch these on/off, or at least provide CSS class names for these items.

Reply:
Yeah, this feels super hacky. The other option was disabling the inner elements that _do_ have classes, but the `border-bottom` is on the wrapping container, so you ended up with an extra thick border under the Google preview button.